community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/cache/idmapping/string/ParallelSort.java

```diff
@@ -258,7 +260,7 @@ public void run()
                 e.printStackTrace();
```

Comment:
Yeah... right. It's a benign failure of sorts. So should we remove stacktrace printing and instead restore interrupted flag here you think?